api/v1alpha1/api.go

```diff
@@ -437,6 +437,15 @@ const (
 	//
 	// https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#api-specs
 	APISchemaAzureOpenAI APISchema = "AzureOpenAI"
+	// APISchemaGeminiVertexAI is the schema followed by Gemini models hosted on GCP's Vertex AI platform.
+	//
+	// https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.endpoints/generateContent?hl=en
+	APISchemaGeminiVertexAI APISchema = "GeminiVertexAI"
```

Comment:
This should be `GCPVertexAI` - this is not addressing my previous comment and thats' what Dan suggested https://github.com/envoyproxy/ai-gateway/pull/635#discussion_r2145246822.